{"name": "test_xfail_with_run_false", "status": "passed", "description": "\n    >>> import pytest\n\n    >>> @pytest.mark.xfail(run=False)\n    ... def test_xfail_with_run_false_example():\n    ...     pass\n    ", "attachments": [{"name": "stdout", "source": "f38d0a13-b8c9-46bd-a532-eb8b5ebef045-attachment.txt", "type": "text/plain"}], "start": 1710872970318, "stop": 1710872970356, "uuid": "494f79f9-3238-4d92-9127-db4239db7956", "historyId": "78740ced9ce372467e44cdfe95c4a75a", "testCaseId": "78740ced9ce372467e44cdfe95c4a75a", "fullName": "tests.allure_pytest.acceptance.status.xfail_setup_status_test#test_xfail_with_run_false", "labels": [{"name": "parentSuite", "value": "tests.allure_pytest.acceptance.status"}, {"name": "suite", "value": "xfail_setup_status_test"}, {"name": "host", "value": "MacBook-Pro-whenessel.local"}, {"name": "thread", "value": "29518-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.allure_pytest.acceptance.status.xfail_setup_status_test"}]}